setup.py

```diff
@@ -15,8 +15,8 @@
     long_description_content_type="text/markdown",
     packages=find_packages(where='.', exclude=['tests.*']),
     install_requires=[
-            "requests == 2.21.0",
-            "uritemplate == 3.0.0",
-            "iso8601 == 0.1.12"
+            "requests >= 2.21.0",
```

Comment:
@nateklaiber We are having conflicts with this requiring a specific version with other dependencies in our project.  This should address that.

Reply:
@stevecox - makes sense and sounds good. I am able to use the python client against my production credentials without issues (navigating around directory, projects, campaigns, templates, etc).